docs/app/image-optimization.md

```diff
@@ -0,0 +1,12 @@
+# Image Optimization
+
+If your project has a requirement for image content that doesn't lend itself to something like SVG, you may want to use the image optimization functionality that NextJS provides in order to better optimize your image(s) for improved performance and UX.
+
+[The NextJS docs](https://nextjs.org/docs/app/building-your-application/optimizing/images) have extensive info on how to use the `<Image>` component, but a couple of items are worth noting:
+
+* When running in production (not when running `next dev`), you will need to ensure that the `sharp` library is installed. 
+
+  ```bash
+  npm i sharp
+  ```
+* Depending on your tech stack, you may need to ensure that the `.next/cache/images` folder is present and write permissions have been applied 
```

Comment:
Wondering if this line is more confusing than helpful tbh. Is there more specifics about what type of tech stack would need this? Would it be an obvious error if this needs applied, or is it important to call out since it could get overlooked when deployed?

Reply:
it("renders correctly based on local feature flag is true", async () => {
      process.env.NEXT_PUBLIC_FEATURE_foo = "true";

      const result = await Controller();
      render(result);

      expect(await screen.findByText(/flag is enabled/i)).toBeInTheDocument();
    });

Reply:
Yeah, I didn't love that line. I added it due to the specific report from the original issue, but obviously that issue showed that the error message that is displayed would likely point to the correct resolution.

Happy to accept edits or just removal!

Reply:
I vote for removing 🔪